packages/labs/cli/src/lib/generate/react.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * @license
+ * Copyright 2022 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import {Package} from '@lit-labs/analyzer/lib/model.js';
+
+export const run = async (analysis: Package, console: Console) => {
```

Comment:
Ok for now, but I think we'll want a `@lit-labs/gen-wrapper-react` package very soon.